Packs/EDL/Integrations/EDL/EDL.yml

```diff
@@ -70,6 +70,20 @@ configuration:
   name: add_comment_if_empty
   required: false
   type: 8
+- additionalinfo: 'CIDRs with prefix size smaller than this value will not be included'
+  defaultvalue: 8
+  display: 'Maximum CIDR prefix size'
```

Comment:
CIDR network prefix bits size

Reply:
I actually like `Maximum CIDR prefix size` a bit better, we can join the two to `Maximum CIDR network prefix bits size`